frontend/src/lib/components/icon/PreviewAllIcons.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
```

Comment:
will you remove this component when the PR is approved? or what is the use of this component?

Reply:
I was going to save it so it can be used in a user guide to easily see all available components.

Reply:
True, I think it would be nice to have a preview directory of all of the components. I'm just wondering if it's better to keep that separate from the impromptu preview of a component within a PR, because we might run into merge issues if there are several PRs open at once and they all try to edit the `_test` route… 🤔